src/workspace-data/ImportStatus.test.ts

```diff
@@ -0,0 +1,217 @@
+    it('polls if import job is still pending', async () => {
+      // Arrange
+      asyncImportJobStore.set([
+        { targetWorkspace: { namespace: 'test-workspaces', name: 'google-workspace' }, jobId: 'workspace-job-1' },
+      ]);
+
+      const getImportJobStatus = jest.fn().mockResolvedValue({ status: 'Pending' });
+      const mockAjax: DeepPartial<AjaxContract> = {
+        Workspaces: {
+          workspace: (_namespace, _name) => ({
+            getImportJobStatus,
+          }),
+        },
+      };
+      asMockedFn(Ajax).mockImplementation(() => mockAjax as AjaxContract);
+
+      // Act
+      render(h(ImportStatus, {}));
+      await act(async () => {
+        jest.advanceTimersByTime(60000);
+      });
+
+      expect(notify).not.toHaveBeenCalled();
+
+      // Should poll again by calling getImportJobStatus again
+      await act(async () => {
+        jest.advanceTimersByTime(60000);
+      });
+
+      expect(getImportJobStatus).toHaveBeenCalled();
+      expect(notify).not.toHaveBeenCalled();
+    });
+    it('notifies error if import failed', async () => {
+      // Arrange
+      asyncImportJobStore.set([
+        { targetWorkspace: { namespace: 'test-workspaces', name: 'google-workspace' }, jobId: 'workspace-job-1' },
+      ]);
+
+      const getImportJobStatus = jest.fn().mockResolvedValue({ status: 'Error', message: 'There has been an error.' });
+      const mockAjax: DeepPartial<AjaxContract> = {
+        Workspaces: {
+          workspace: (_namespace, _name) => ({
+            getImportJobStatus,
+          }),
+        },
+      };
+      asMockedFn(Ajax).mockImplementation(() => mockAjax as AjaxContract);
+
+      // Act
+      render(h(ImportStatus, {}));
+      await act(async () => {
+        jest.advanceTimersByTime(60000);
+      });
+
+      // Assert
+      expect(notify).toHaveBeenCalledWith('error', 'Error importing data.', {
+        notificationMessage: 'There has been an error.',
+      });
+    });
+    it('notifies success when import completes', async () => {
+      // Arrange
+      asyncImportJobStore.set([
+        { targetWorkspace: { namespace: 'test-workspaces', name: 'google-workspace' }, jobId: 'workspace-job-1' },
+      ]);
+
+      const getImportJobStatus = jest.fn().mockResolvedValue({ status: 'Done' });
+      const mockAjax: DeepPartial<AjaxContract> = {
+        Workspaces: {
+          workspace: (_namespace, _name) => ({
+            getImportJobStatus,
+          }),
+        },
+      };
+      asMockedFn(Ajax).mockImplementation(() => mockAjax as AjaxContract);
+
+      // Act
+      render(h(ImportStatus, {}));
+      await act(async () => {
+        jest.advanceTimersByTime(60000);
+      });
+
+      // Assert
+      expect(notify).toHaveBeenCalledWith(
+        'success',
+        'Data imported successfully.',
+        expect.objectContaining({
+          //   message: expect.stringContaining('test-workspaces / google-workspace'),
```

Comment:
Help!  The `message` item is a `React.Fragment` which has the given text in a `<p>` inside it.  I could not figure out how to test for that.  Suggestions?

Reply:
Instead of importing `render` from `@testing-library/react`, use:

```ts
import { renderWithAppContexts as render } from 'src/testing/test-utils';
```

Reply:
🦸